tpl/encoding/encoding.go

```diff
@@ -50,9 +50,13 @@ func (ns *Namespace) Base64Encode(content interface{}) (string, error) {
 	return base64.StdEncoding.EncodeToString([]byte(conv)), nil
 }
 
-// Jsonify encodes a given object to JSON.
-func (ns *Namespace) Jsonify(v interface{}) (template.HTML, error) {
-	b, err := json.Marshal(v)
+// Jsonify encodes a given object to JSON, returning pretty printed output.
+func (ns *Namespace) Jsonify(v interface{}, indentSpacing string) (template.HTML, error) {
+	if indentSpacing == "" {
+		indentSpacing = "  "
+	}
+
+	b, err := json.MarshalIndent(v, "", indentSpacing)
```

Comment:
Getting some very weird behaviour here - when I switched to `json.MarshalIndent` and updated the respective table test, there was a panic occurring. I was looking into the docs for `template.HTML` but everything looked ok at first glance.

Reply:
Respective logs from Travis: https://travis-ci.org/gohugoio/hugo/jobs/444259677#L759

Reply:
Currently running on go version go1.11.1 darwin/amd64 if that helps

Reply:
**Update**: `template.HTML("[\n  \"a\",\n  \"b\"\n]")` was the answer - I had to escape quotes